build-logic/documentation/src/main/groovy/gradlebuild/docs/FindBrokenInternalLinks.java

```diff
@@ -62,12 +64,19 @@ public abstract class FindBrokenInternalLinks extends DefaultTask {
     @PathSensitive(PathSensitivity.RELATIVE)
     public abstract DirectoryProperty getJavadocRoot();
 
+    @InputDirectory
+    @PathSensitive(PathSensitivity.RELATIVE)
+    public abstract DirectoryProperty getReleaseNotesRoot();
+
     @OutputFile
     public abstract RegularFileProperty getReportFile();
 
     @TaskAction
     public void checkDeadLinks() {
         Map<File, List<Error>> errors = new TreeMap<>();
+        File releaseNotesFile = getReleaseNotesRoot().file("raw.html").get().getAsFile();
```

Comment:
❓ Shouldn't this name be wired externally? Currently, it's configured here
* https://github.com/gradle/gradle/blob/a1531c5f1fdf32ce0a1863f54bf23d57beee5682/build-logic/documentation/src/main/groovy/gradlebuild/docs/GradleReleaseNotesPlugin.java#L59

Reply:
Changed to RegularFileProperty input